### DISCUSSION TITLE
I want to add notes to orders that are sent through from the users input, how can I achieve this?

### DISCUSSION THREAD

**Discussion by giovihh** [medusajs/medusa#4167] [Feature Requests]
I have products that I need more information from customers and I would like to add order notes during checkout (and of course view them from the admin)

**Comment by srindom** [comment]
One option is to use Cart metadata for user notes - the metadata is copied to the resulting order, and you can view it in the order data.

The same approach can be used to associate a note with a specific item - in this case, you would pass the metadata on the LineItem when adding an item to the cart. 

Can you share more details on what you plan to use this for?
